app/atoms/ln-patternlab/_ln-m-widget-contact.scss

```diff
@@ -0,0 +1,26 @@
+$_phone_prefix: 'T';
```

Comment:
I put them in the atoms folder because we have all basic styles there. I think that having the syles for all ln-pattenlab components in only one folder will be very useful for development as it will be easy to find a style file for a generic component that must be customized. Maybe we can move ln-patternlab styles folder to the app base...